packages/rainbowkit/src/wallets/walletConnectors/tallyHoWallet/tallyHoWallet.ts

```diff
@@ -0,0 +1,46 @@
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface InjectedWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+declare global {
+  interface Window {
+    tally: any;
+  }
+}
+
+export const tallyHoWallet = ({
+  chains,
+  shimDisconnect,
+}: InjectedWalletOptions): Wallet => ({
+  createConnector: () => {
+    return {
+      connector: new InjectedConnector({
+        chains,
+        options: {
+          getProvider() {
+            const getTally = (tally?: any) =>
+              tally?.isTally ? tally : undefined;
+            if (typeof window === 'undefined') return;
+            return getTally(window.tally);
+          },
+          name: 'Tally',
+          shimDisconnect,
+        },
+      }),
+    };
+  },
+  downloadUrls: {
+    browserExtension:
+      'https://chrome.google.com/webstore/detail/tally-ho/eajafomhmkipbjmfmhebemolkcicgfmd',
+  },
+  iconBackground: '#d08d57',
+  iconUrl: async () => (await import('./tallyHoWallet.svg')).default,
+  id: 'tallyHoWallet',
+  installed: true,
```

Comment:
Installed should have logic to detect if the provider is injected, or not. Reference the MetaMask example: https://github.com/rainbow-me/rainbowkit/blob/78f6cbea363e89ceffa4132b084f14ae90bbf825/packages/rainbowkit/src/wallets/walletConnectors/metaMaskWallet/metaMaskWallet.ts#L56

Reply:
Bumping this; last implementation issue

Reply:
addressed this in the latest commit